device-base/generate-dockerfile.sh

```diff
@@ -127,12 +128,16 @@ for device in $devices; do
 		baseImage='armv7hf-debian'
 		alpine_template='Dockerfile.alpine.tpl'
 		alpine_baseImage='armhf-alpine'
+		fedora_template='Dockerfile.fedora.tpl'
+		fedora_baseImage='armhf-fedora'
 	;;
 	'artik10')
 		template='Dockerfile.tpl'
 		baseImage='armv7hf-debian'
 		alpine_template='Dockerfile.alpine.tpl'
 		alpine_baseImage='armhf-alpine'
+		fedora_template='Dockerfile.fedora.tpl'
+		fedora_baseImage='armhf-fedora'
```

Comment:
I expect this base image will work on armv6hf device (the same as alpine base image) but just test on a RPI model B+ and it doesn't work. So I will update this to armv7hf